ads/google/doubleclick.js

```diff
@@ -87,6 +117,80 @@ function doubleClickWithGpt(global, data, gladeExperiment) {
   container.style.right = '';
   container.style.transform = 'translate(-50%, -50%)';
 
+  // Get multi-size ad request data, if any, and validate it in the following
+  // ways: Ensure that the data string is a comma-separated list of sizes of the
+  // form wxh; that each secondary dimension is strictly less than its primary
+  // dimension counterpart; and, if data-mutli-size-validation is not set to
+  // false, that each secondary dimension is at least 2/3rds of its primary
+  // dimension counterpart.
+  const multiSizeDataStr = data.multiSize || null;
+  if (multiSizeDataStr) {
+    const sizesStr = multiSizeDataStr.split(',');
+    sizesStr.forEach(sizeStr => {
+
+      const size = sizeStr.split('x');
+
+      // Make sure that each size is specified in the form val1xval2.
+      if (!Array.isArray(size) || size.length != 2) {
```

Comment:
I'm not sure. Array.isArray(5) == Array.isArray('i am a string') == false == !Array.isArray([]), as far as I can tell.